perception/lidar_centerpoint/README.md

```diff
@@ -80,6 +80,175 @@ ros2 launch lidar_centerpoint single_inference_lidar_centerpoint.launch.xml pcd_
 `lidar_centerpoint` generates a `ply` file in the provided `detections_path`, which contains the detections as triangle meshes.
 These detections can be visualized by most 3D tools, but we also integrate a visualization UI using `Open3D` which is launched alongside `lidar_centerpoint`.
 
+## Training CenterPoint Model and Deploying to the Autoware
+
+### Overview
+
+This guide provides instructions on training a CenterPoint model using the **mmdetection3d** repository
+and seamlessly deploying it within Autoware.
+
+### Installation
+
+#### Install prerequisites
+
+**Step 1.** Download and install Miniconda from the [official website](https://mmpretrain.readthedocs.io/en/latest/get_started.html).
+
+**Step 2.** Create a conda virtual environment and activate it
+
+```bash
+conda create --name train-centerpoint python=3.8 -y
+conda activate train-centerpoint
+```
+
+**Step 3.** Install PyTorch
+
+Please ensure you have PyTorch installed, and compatible with CUDA 11.6, as it is a requirement for current Autoware.
+
+```bash
+conda install pytorch==1.13.1 torchvision==0.14.1 pytorch-cuda=11.6 -c pytorch -c nvidia
+```
+
+#### Install mmdetection3d
+
+**Step 1.** Install MMEngine, MMCV, and MMDetection using MIM
+
+```bash
+pip install -U openmim
+mim install mmengine
+mim install 'mmcv>=2.0.0rc4'
+mim install 'mmdet>=3.0.0rc5, <3.3.0'
```

Comment:
@kaancolak 

Can you add dockerfile to make this environment in https://github.com/autowarefoundation/mmdetection3d.git?
We can make installation step shorter.

Reply:
@Shin-kyoto -san, thank you for your advice!

Updated the current docker file with these installation steps. 

https://github.com/autowarefoundation/mmdetection3d/pull/1/files#diff-f34da55ca08f1a30591d8b0b3e885bcc678537b2a9a4aadea4f190806b374ddcL1

Reply:
@kaancolak 
Thank you so much!! Can you update this document to align with the environment setup procedures using Docker?

Reply:
@Shin-kyoto sure, I added instructions to here:
https://github.com/autowarefoundation/autoware.universe/pull/5570/files#diff-ef509aa43435872925d0134c14c088da6f51549d59e84cc5bd3e74fd4fa333f9R134